samples/house_price_estimates/README.md

```diff
@@ -0,0 +1,109 @@
+## Prerequisits
+
+* [get_data_endpoints prerequisits and setup](../../get_data_endpoints/README.md)
+
+## Usage
+
+The sample provided assumes that you have put the [training](./data/trainwithpi.csv) and [testing](./data/testwithpi.csv) data sets in some type of data store, for which you have the endpoint and credentials.  The data will be read via arrow-flight protocol.
+
+This example also assumes that Fybrik is configured to use [katalog](https://fybrik.io/v1.0/reference/katalog/) as a data catalog, and OPA is configured as Fybrik's data governance policy manager.  These are the defaults in [Fybrik's quick start](https://fybrik.io/v1.0/get-started/quickstart/)
+
+### Prepare Data Assets and Storage
+
+Edit the following files to include the relevant details of where your data is stored.  You may use an object store of your choice such as AWS S3, IBM Cloud Object Storage or Ceph.
+
+* house-price-demo-secret.yaml
+* testpii-asset.yaml
+* trainpii-asset.yaml
+
+Register in the Data Catalog (katalog) the training and testing data to be used by the pipeline, noting the catalog ID of each.
+
+```bash
+kubectl apply -f data/house-price-demo-secret.yaml -n kubeflow
+kubectl apply -f data/testpii-asset.yaml -n kubeflow
+kubectl apply -f data/trainpii-asset.yaml -n kubeflow
+```
+
+### Register Governance Policies
+
+Run the following commands to register the data governance policies in the [OPA governance engine being used by Fybrik](https://fybrik.io/v1.0/tasks/using-opa/):
+
+The read policy dictates that columns marked as having personal information should be redacted.
+
+Read Policy:
+
+```bash
+kubectl -n fybrik-system create configmap pii-read-policy --from-file=data/pii-read-policy.rego
+		
+kubectl -n fybrik-system label configmap pii-read-policy openpolicyagent.org/policy=rego
+		
+while [[ $(kubectl get cm pii-read-policy -n fybrik-system -o 'jsonpath={.metadata.annotations.openpolicyagent\.org/policy-status}') != '{"status":"ok"}' ]]; do echo "waiting for policy to be applied" && sleep 5; done
+```
+
+The write policy allows all data to be written to any location registered with Fybrik as per the [prequisits](../../get_data_endpoints/README.md).
+
+Write Policy:
+
+```bash
+kubectl -n fybrik-system create configmap allow-write-policy --from-file=data/allow-write-policy.rego
+		
+kubectl -n fybrik-system label configmap allow-write-policy openpolicyagent.org/policy=rego
+		
+while [[ $(kubectl get cm allow-write-policy -n fybrik-system -o 'jsonpath={.metadata.annotations.openpolicyagent\.org/policy-status}') != '{"status":"ok"}' ]]; do echo "waiting for policy to be applied" && sleep 5; done
+
+```
+
+### Upload Pipeline
+
+Upload either the pipeline-argo.yaml or pipeline-tekton.yaml file via the Kubeflow Pipeline GUI, depending whether you installed kubeflow pipelines with Argo (default) or with Tekton Pipelines.
+
+See slide 17 of the [demo presentation](https://drive.google.com/file/d/1xn7pGe5pEAEZxnIzDdom9r7K6s78alcP/view?usp=sharing).
```

Comment:
please mention that the pipeline definition files are under `samples/house_price_estimates/data`